resources/views/adm/mship/feedback/view.blade.php

```diff
@@ -2,132 +2,179 @@
-                </div>
-                @endif
-              <div class="col-lg-12 col-md-12">
-                <b>Actioned Comment:</b></br>
-                {{ $feedback->actioned_comment }}
-              </div>
-              @else
-                @if (\Auth::user()->hasChildPermission('adm/mship/feedback/view/*/action'))
-                  {{ Form::open(['route' => ['adm.mship.feedback.action', $feedback->id]]) }}
-                  <div class="col-md-9">
-                    <p>
-                      {{ Form::label('Action Comment')}}
-                      {{ Form::textarea('comment', null, ['rows' => 1, "cols"=> 10]) }}
-                    </p>
-                    {{ Form::submit('Mark Actioned', ['class' => 'btn btn-warning']) }}
-                  </div>
-                @endif
-              @endif
-          </div>
-      </div>
+        </div>
     </div>
-</div>
 
-<!-- Main row -->
-<div class="row">
-    <!-- Left col -->
-    <section class="col-lg-12">
-        <div class="box box-primary">
-            <div class="box-header">
-                <h3 class="box-title ">
-                    Questions &amp; Answers
-                </h3>
-            </div><!-- /.box-header -->
-            <div class="box-body">
-              <div class="row">
-                @foreach ($feedback->answers()->notPermanent()->get() as $answer)
-                  <div class="col-md-4">
-                    <div class="box box-warning">
-                        <div class="box-header">
-                            <h4 class="box-title ">
-                                {{ $answer->question()->withTrashed()->first()->question }}
-                            </h4>
-                        </div><!-- /.box-header -->
-                        <div class="box-body">
-                            {!! nl2br(e($answer->response)) !!}
+    @if (\Auth::user()->hasChildPermission('adm/mship/feedback/view/*/action'))
+
+        <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="box box-primary">
+                    <div class="box-header">
+                        <h3 class="box-title ">
+                            Send To User
+                        </h3>
+                    </div>
+
+                    <div class="box-body">
+                        <div class="row">
+                            <div class="col-md-12">
+                                @if (!$feedback->sent_at)
+                                {{ Form::open(['route' => ['adm.mship.feedback.send', $feedback->id]]) }}
+                                <div class="col-md-12 text-center">
+                                    <p>
+                                        {{ Form::label('Comment To User')}}
+                                        {{ Form::textarea('comment', null, ['rows' => 4, "cols"=> 10, "class" => "form-control"]) }}
```

Comment:
Possibly add a "helper" text here, saying that if left blank, it will advise the member that positive feedback was left about them.

Reply:
99a20e3